### PR TITLE
Add Food

### PR DESCRIPTION
**Pull request by mkhajure7000**

### CODE REVIEW

File: app/controllers/foods_controller.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
+class FoodsController < ApplicationController
+  before_action :get_restaurant
+  before_action :set_food, only: %i[edit update destroy]
+  def index
```
Comment (Shubham-Porwall) [inline app/controllers/foods_controller.rb:5]: fix spaces

File: app/controllers/foods_controller.rb [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    @foods = @restaurant.foods.all
+  end
+
+  def new
+    @food = Food.new
+  end
+  
+  def create
+    @food = @restaurant.foods.new(food_params)
+    if @restaurant.save
```
Comment (Shubham-Porwall) [inline app/controllers/foods_controller.rb]: food.save?

File: app/views/foods/edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<h1>Edit Dish</h1>
+<%= render "form",food: @food %>
```
Comment (Shubham-Porwall) [inline app/views/foods/edit.html.erb]: Fix spaces

File: app/views/foods/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+<h1>Create Dish</h1>
+<%= render "form",food: @food %>
```
Comment (Shubham-Porwall) [inline app/views/foods/new.html.erb]: spaces

File: app/controllers/foods_controller.rb [hunk 5]
```diff
@@ -0,0 +1,59 @@
+class FoodsController < ApplicationController
+  before_action :get_restaurant
```
Comment (Shubham-Porwall) [inline app/controllers/foods_controller.rb]: Specify for which methods you need this filter

**Review by Shubham-Porwall — APPROVED** [review]
Last Few Changes !

File: app/controllers/foods_controller.rb [hunk 6]
```diff
@@ -0,0 +1,59 @@
+class FoodsController < ApplicationController
+  before_action :get_restaurant, only: %i[index new create edit update destroy]
```
Comment (Shubham-Porwall) [inline app/controllers/foods_controller.rb:2]: DId you need this action for new ?

Comment (mkhajure7000) [reply app/controllers/foods_controller.rb:2]: yes

**Pull request merged**